### PR TITLE
Make GPQueue and Controller Scheme useful!

### LINKED ISSUE #171: Make controllerschemes and gpQueues more usable
opened on behalf of @jellegerbrandy 
The new `gpQueue`, `ucontrollerScheme`, `controllerScheme` and `ControllerOrga
-nization`
are all free-standing entities, not related to the other graph entities at all.

This makes them difficult and expensive to use.

In particular, what we need is:

* `Proposal.gpQueue`: the `gpQueue` that the proposal is part of **<- plz make this first**
* `DAO.controllerSchemes`
* `DAO.uControllerSchemes`
* (given that the DAO either uses controller and ucontroller (and never both, is that correct?), it would simplify the data model is there is just a singe "controllerScheme" entity
instead of a controllerScheme and a uControllerScheme with exactly the same data model, so we can have just one "controllerScheme" entity that
contains data either entity...)
* `DAO.gpQueues`: the gpQueues of these DAO

### PR DESCRIPTION
**Pull request by ben-kaufman**
Fix #171

### CODE REVIEW

File: src/domain/proposal.ts [hunk 1]
```diff
@@ -134,6 +134,13 @@ export function updateGPProposal(
   proposal.organizationId = gpProposal.value0;
   proposal.expiresInQueueAt = timestamp.plus(params.value1);
   proposal.createdAt = timestamp;
+  updateThreshold(
+    proposal.dao.toString(),
+    gp.threshold(proposal.paramsHash, proposal.organizationId),
```
Comment (orenyodfat) [inline src/domain/proposal.ts]: proposal.organizationId seems to be redundant. as it is the id of GPQUE.

File: test/integration/SchemeRegistrar.spec.ts [hunk 2]
```diff
@@ -171,7 +171,7 @@ describe('SchemeRegistrar', () => {
 
         const executedIsIndexed = async () => {
           return (await sendQuery(getSchemeRegistrarProposalExecuteds)).schemeRegistrarProposalExecuteds.length
-           > prevExecutedsLength;
+           > prevExecutedsLength + 1;
```
Comment (orenyodfat) [inline test/integration/SchemeRegistrar.spec.ts]: why and how this pr effect getSchemeRegistrarProposalExecuteds length ?

Comment (ben-kaufman) [reply test/integration/SchemeRegistrar.spec.ts]: That's a bug I fixed is not exactly related.

File: src/domain/proposal.ts (hunk 1 above)
Comment (ben-kaufman) [reply src/domain/proposal.ts]: It causes problems to remove it and Alchemy might be using it.

**Pull request merged**